src/traces/scatterpolar/hover.js

```diff
@@ -42,26 +42,26 @@ function makeHoverPointText(cdi, trace, subplot, pointData) {
     radialAxis._hovertitle = 'r';
     angularAxis._hovertitle = 'θ';
 
+    var rVal = radialAxis.c2l(cdi.r);
+    pointData.rLabel = Axes.tickText(radialAxis, rVal, 'hover').text;
+
+    // N.B here the ° sign is part of the formatted value for thetaunit:'degrees'
+    var thetaVal = angularAxis.thetaunit === 'degrees' ? Lib.rad2deg(cdi.theta) : cdi.theta;
+    pointData.thetaLabel = Axes.tickText(angularAxis, thetaVal, 'hover').text;
```

Comment:
This here is an interesting case. Consistently, note also that:

```js
Plotly.newPlot(gd, [{
  y: [1],
  hovertemplate: '%{y}'
}], {
  yaxis: {ticksuffix: ' --', tickprefix: '-- '}
})
```

shows `-- 1 --` in the hover labels.

----

Please let me know if you think `ticksuffix` and `tickprefix` should be left out.